x/merkledb/db.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	DefaultEvictionBatchSize = 100
+	DefaultEvictionBatchSize = 256 * units.KiB
```

Comment:
Seems like we may want bigger than this?

Reply:
This is just the default used in `getStandaloneTrieView` which is only used in proof verification

Reply:
That said we can make it bigger. 1 MiB?

Reply:
I was thinking 1MB is a good place to start.

Reply:
Why would we want larger than this? This isn't a cache - this size that is used to evict from the cache... As dan said - this is only used (internally) for writing during proof verification... which is writing into a `memdb`

Reply:
welp I already changed it to 1 MiB. In any case I don't think it matters much. I also unexported it since this is only used in this package.

Reply:
More I think about this - I think the value should be 0.